File: app/src/main/java/com/jeanpigomez/codingtest/network/LanguageApi.java
package com.jeanpigomez.codingtest.network;

import com.jeanpigomez.codingtest.model.Language;

import java.util.List;

import io.reactivex.Flowable;
import retrofit2.http.GET;

public interface LanguageApi {

    @GET("/items")
    Flowable<List<Language>> getLanguageList();
}

File: app/src/main/java/com/jeanpigomez/codingtest/network/LanguageApi.kt
package com.jeanpigomez.codingtest.network

import com.jeanpigomez.codingtest.model.Language
import io.reactivex.Observable
import retrofit2.http.GET

interface LanguageApi {

    @GET("/items")
    fun getLanguages(): Observable<List<Language>>
}
